tests/drivers/bbram/testcase.yaml

```diff
@@ -13,3 +13,8 @@ tests:
   drivers.bbram.npcx:
     extra_args: CONF_FILE="npcx.conf"
     filter: dt_compat_enabled("nuvoton,npcx-bbram")
+  drivers.bbram.stm32:
+    platform_allow: stm32f0_disco stm32f3_disco stm32f4_disco stm32f723e_disco
+                    stm32h735g_disco stm32l1_disco stm32l476g_disco
+                    nucleo_wb55rg
```

Comment:
When technically possible, avoid use of `platform_allow`.
In this case, why not enabling bbram on those boards and filter on `dt_compat_enabled()` as done on other platforrms ?

Reply:
> When technically possible, avoid use of platform_allow.

OK. Understood.

> In this case, why not enabling bbram on those boards and filter on dt_compat_enabled() as done on other platforrms ?

Done